OracleManagementAgent/README.md

```diff
@@ -112,50 +134,105 @@ Users can provide custom shell script commands to execute before starting Manage
 1. Follow the steps to build and run a container and validate the output of `init-agent.sh` script is visible in the logs by running the following command
 
     ```shell
+    # inspect container output logs
     $ docker logs mgmtagent-container
     ```
 
-#### Helpful administration commands
+## Troubleshooting
+
+Solutions to common issues when running Management Agent in a container are described in the following steps
+
+1. mkdir: cannot create directory '/opt/oracle/bootstrap': Permission denied
+
+    * Ensure the mounted volume exists and is accessible by the user used to run the container
+    * Verify the user USERID and GROUPID used match the permissions set on the mounted bind volume
+    * Verify the mounted bind volume exists on the host
+
+1. Invalid argument: /opt/oracle/mgmtagent_secret/input.rsp
+
+    * Ensure the install key exists at the required location
+
+1. Management Agent registration failures due to old state files from a prior install
+<!-- markdownlint-disable MD013 -->
+<!-- markdownlint-disable MD046 -->
+    * Once a Management Agent instance is deregistered that instance must be shutdown and any associated state files must be removed from the filesystem. Starting a deregistered Management Agent instance again can result in unregistered agent failures. This situation can present itself when old state files from a prior installation are present on the filesystem and made available to a new Management Agent container deployment. Run the command given below on the host filesystem to perform the necessary cleanup on the bind mount location and perform the deployment again starting at the prerequisites step.
+
+    ```shell
+    # cleanup old files from prior installation
+    $ rm -rf /home/$USER/oracle/
+    ```
+<!-- markdownlint-enable MD046 -->
+<!-- markdownlint-enable MD013 -->
+
+## Helpful commands
 
 1. Starting a stopped Management Agent Container
 
     ```shell
+    # start container and agent
     $ docker start mgmtagent-container
     ```
 
 1. Stopping a running Management Agent Container
 
     ```shell
+    # stop container and agent
```

Comment:
Remove this comment as it's just repeating the instruction above.

Reply:
will remove